### ISSUE TITLE
are API exposed?

### ISSUE THREAD

**Issue by getinfo-developer** [Attendize/Attendize#467]
Hi,  It is really good that this product have QR code scanner. But sometime it is easier to checked in using native iOS & Android app. I am planning to create a simple QR scanning app for check in .  Do we have API exposed to consume to check in?

**Comment by Luid101** [comment]
Yeah, I am also curious to know if any API's are exposed. I already have a front end shop that can sell tickets, and I am just interested in using Attendize to manage who is going to what event and sign them in at the door.

**Comment by jeremyquinton** [comment]
@getinfo-developer @Luid101 currently Attendize is an MVC application and does have an API. It would make sense to add an API at some point.

**Comment by Bellusterra** [comment]
This would also give the ability to list all events on a server in one location and to be able to sort events by location correct?

**Comment by RayhanYulanda** [comment]
@getinfo-developer @Luid101 @jeremyquinton @Ezyweb-uk @Bellusterra i just made mobile scanner with flutter framework for attendize check in. check my repo and give me some love ❤️💙💚💜💛😄
https://github.com/RayhanYulanda/Attendize-Scanner-Mobile
